box.json

```diff
@@ -0,0 +1,29 @@
+{
+    "alias": "satis.phar",
+    "main": "bin/satis",
+    "output": "satis.phar",
+    "stub": true,
+    "directories": [ "src", "views" ],
+    "files": [
+        "LICENSE"
+    ],
+    "finder": [{
+        "name": "*.php",
```

Comment:
The `Compiler` class does not include those either. So I'm guessing they're either never necessary or people rarely/never use satis as a Phar.